File: server/src/main/java/com/example/projectstore/issue/IssueDtoSimple.java
package com.example.projectstore.issue;

import com.example.projectstore.user.UserDtoSimple;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
@AllArgsConstructor
public class IssueDtoSimple {
    private UUID id;
    private UserDtoSimple createdBy;
    private String title;
    private Boolean isOpen;
    private LocalDateTime created;

}

File: app/src/lib/models/issue/IssueDtoSimple.ts
import type { UserDtoSimple } from "../user/UserDtoSimple"

type IssueDtoSimple = {
    id: string
    createdBy: UserDtoSimple
    title: string
    isOpen: boolean
    projectId: string
    created: string
}

export type {IssueDtoSimple}